File: EDoctorsApp/EDoctors/app/src/main/java/com/incubation_lab/edoctors/DoctorLoginFragment.java
package com.incubation_lab.edoctors;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.viewpager.widget.ViewPager;

import com.google.android.material.tabs.TabLayout;

public class DoctorLoginFragment extends Fragment {
    public static Fragment newInstance() {
        return new DoctorLoginFragment();
    }

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {

        View root = inflater.inflate(R.layout.fragment_doctor_login, container, false);

        return root;
    }
}

File: EDoctorsApp/app/src/main/java/com/incubation_lab/edoctor/DoctorLoginFragment.kt
package com.incubation_lab.edoctor

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class DoctorLoginFragment : Fragment() {


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {

        return inflater.inflate(R.layout.fragment_doctor_login, container, false)
    }

    companion object {

        @JvmStatic
        fun newInstance() =
                DoctorLoginFragment()
    }
}